libs/util/StringUtils.arucas

```diff
@@ -75,4 +75,110 @@ class StringBuilder {
     fun toString() {
         return this.string;
     }
-}
\ No newline at end of file
+}
+class StringUtil{
+    /**
```

Comment:
I just realised that this whole class is duplicated.
The FormatUtils class should be removed. And all of the issues should be instead applied here